manageiq-operator/pkg/apis/manageiq/v1alpha1/manageiq_types.go

```diff
@@ -14,284 +14,288 @@ type ManageIQSpec struct {
 
 	// Application name used for deployed objects (default: manageiq)
 	// +optional
-	AppName string `json:"appName"`
+	AppName string `json:"appName,omitempty"`
```

Comment:
Can we get these in a separate PR?  They seem unrelated.

Reply:
Done and rebased